### ISSUE TITLE
Stuck at 100% CPU when analyzing SciPy 1.2.1

### ISSUE THREAD

**Issue by mhzed** [Microsoft/pyright#66]
**Describe the bug**
Pyright (1.0.9) gets stuck on 100% CPU when analyzing SciPy 1.2.1 package.

**To Reproduce**
Procedure 1:
```bash
npm i -g pyright
cd <scipy_1.2.1_site_package_path>
pyright .
```
Procedure 2:
Create a pipenv project, and
```bash
pipenv install scipy
```
Open project in VsCode.  Import scipy in a python file.
```python
import scipy.stats
```
F12 or 'go to definition' to trigger pyright analysis.
In OSX Activity monitor, "Code Helper" CPU usage stays at 90+%.

**Expected behavior**
Pyright stops with analysis results.

**Screenshots or Code**
See above.

**VS Code extension or command-line**
Both command line and plugin.

**Additional context**

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version (1.0.11).

**Comment by erictraut** [comment]
This is fixed in 1.0.11, which I just released.
